src/Routing/RouteBuilder.php

```diff
@@ -149,14 +149,24 @@ public function routeClass($routeClass = null)
      * extensions applied. However, setting extensions does not modify existing routes.
      *
      * @param null|string|array $extensions Either the extensions to use or null.
+     * @param boolean $merge If TRUE `$extensions` will be merged with current
      * @return array|null
      */
-    public function extensions($extensions = null)
+    public function extensions($extensions = null, $merge = true)
```

Comment:
The collection got a merge flag as I had hoped to replace Router::extensions() implementation entirely. That didn't happen because the default extensions needed to be tracked separately for a reason I can no longer remember.

I also didn't see a use case for redeclaring the extensions within a single scope block originally. I had assumed that people would define all their extensions in a scope at the same time.